group_test.go

```diff
@@ -432,3 +433,29 @@ func TestDeleteGroup(t *testing.T) {
 		t.Logf("Deleted group with id: %d", id)
 	}
 }
+
+func TestEnableStreamingGroup(t *testing.T) {
+	bridge := New(hostname, username)
+	id := 1
+	group, err := bridge.GetGroup(id)
+	if err != nil {
+		t.Fatal(err)
+	}
+	err = group.EnableStreaming()
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+func TestDisableStreamingGroup(t *testing.T) {
+	bridge := New(hostname, username)
+	id := 1
+	group, err := bridge.GetGroup(id)
+	if err != nil {
+		t.Fatal(err)
+	}
+	err = group.DisableStreaming()
+	if err != nil {
+		t.Fatal(err)
+	}
+}
```

Comment:
Can you also make sure to mock the Stream API in `huego_test.go`

Reply:
I have some of the streaming data included in the group mock requests. I can double check